package.json

```diff
@@ -1,13 +1,30 @@
 {
   "name": "grist-plugin-shopify",
-  "version": "1.0.0",
+  "version": "0.0.1",
+  "description": "Grist plugin to import orders from a Shopify store",
+  "scripts": {
+    "build": "rm -rf dist/ && tsc",
+    "watch": "rm -rf dist/ && npm run tsc-w",
+    "tsc-w": "tsc -w --preserveWatchOutput",
+    "test": "mocha test/**/*.ts"
+  },
+  "license": "Apache-2.0",
+  "files": [
+    "manifest.yml",
+    "dist/**/*.js",
+    "static/"
```

Comment:
Oops, it should've said `frontend/`. Fixed. This would only matter for publishing plugins (e.g. if published via npm), so it's not meaningful yet, but the intent is to document for ourselves which files should matter for publishing, i.e. which files are needed for the plugin to work.